internal/cli/config/default_editor_windows.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+const defaultEditor = "notepad"
```

Comment:
[q] For my own understanding, why are we adopting this type of constant dependencies in another package? I was under the impression we should prefer adding constant close to the files + avoid dependencies (like we don't like usage.go)

Reply:
Thanks! :D